python/health-check/src/health_check/config.py

```diff
@@ -0,0 +1,101 @@
+"""
+Module that contains functionality related to reading `config.toml`
+and getting paths used for configuration, templating, or building
+containers
+"""
+
+import functools
+import os
+from typing import Any, Dict, List
+import tomli
+from pathlib import Path
+import json
+import jinja2
+
+BASE_DIR = os.path.dirname(os.path.abspath(__file__))
+CONFIG_DIR = os.path.join(BASE_DIR, "config")
+TEMPLATES_DIR = os.path.join(CONFIG_DIR, "templates")
+CONTAINERS_DIR = os.path.join(BASE_DIR, "containers")
+CONFIG_TOML_PATH = os.environ.get(
+    "HEALTH_CHECK_TOML", os.path.join(BASE_DIR, "config.toml")
+)
+
+
+@functools.lru_cache
+def _init_jinja_env() -> jinja2.Environment:
+    return jinja2.Environment(loader=jinja2.FileSystemLoader(TEMPLATES_DIR))
+
+
+@functools.lru_cache
+def parse_config() -> Dict:
+    if not os.path.exists(CONFIG_TOML_PATH):
+        raise ValueError(f"Config file does not exist: {CONFIG_TOML_PATH}")
+
+    with open(CONFIG_TOML_PATH, "rb") as f:
+        conf = tomli.load(f)
+    return conf
+
+
+def get_json_template_filepath(json_relative_path: str) -> str:
+    return os.path.join(TEMPLATES_DIR, json_relative_path)
+
+
+def load_jinja_template(template: str) -> jinja2.Template:
+    return _init_jinja_env().get_template(template)
+
+
+def load_dockerfile_dir(dockerfile_dir: str) -> str:
```

Comment:
non-blocking, we should unify these... `load` has a different meaning in this context.. Of course if this is accepted, we need to refactor all the method calls too.

Looking briefly at the code, we might be able to delete this method. We should double check whether there are methods not used now that we don't build containers.

```suggestion
def get_dockerfile_dir(dockerfile_dir: str) -> str:
```